cdb2api/cdb2api.c

```diff
@@ -304,8 +306,14 @@ void (*cdb2_uninstall)(void) = CDB2_UNINSTALL_LIBS;
 
 #if WITH_DL_LIBS
 #include <dlfcn.h>
-void cdb2_set_install_libs(void (*ptr)(void)) { cdb2_install = ptr; }
-void cdb2_set_uninstall_libs(void (*ptr)(void)) { cdb2_uninstall = ptr; }
+    void cdb2_set_install_libs(void (*ptr)(const char *))
```

Comment:
I think we should maybe update the functions [here](https://github.com/bloomberg/comdb2/blob/32e91091f5a2e24c54457071eaa7258068ca5137/tests/tools/api_libs.c#L6).

Might make sense to put declarations for these functions into `cdb2api_int.h` and just include that in `api_libs.c` to avoid duplication.

Reply:
Tru